prow/plugins.yaml

```diff
@@ -93,6 +94,11 @@ plugins:
   google/cadvisor:
   - trigger
 
+  google/kubeflow:  
```

Comment:
@cjwagner will we get hooks from this repo without specifically configuring it? (thinking not)

Reply:
I would also guess not. Hooks can be configured at the repo or org scope so it is possible that we already get hooks for all repos in the google org, but I would guess that they are just enabled for `google/cadvisor` as the entire google org would be excessive.

A repo owner will likely need to set up webhooks for the kubeflow repo via the Github repo settings. You can coordinate with me to get the HMAC secret.

Reply:
I already configured webhooks for google/kubeflow and github shows they are being delivered to https://prow.k8s.io/hook. 

You already shared the HMAC secret with me when I setup prow for tensorflow/k8s.